buildspec_integ.yml

```diff
@@ -9,7 +8,11 @@ phases:
       - mkdir -p /go/src/github.com/aws/amazon-ecs-cli
       - cp -R . /go/src/github.com/aws/amazon-ecs-cli/
       - cd /go/src/github.com/aws/amazon-ecs-cli/
+      - echo "installing dependencies..."
+      - go get github.com/wadey/gocovmerge
+      - echo "building test binary..."
+      - go test -coverpkg ./ecs-cli/modules/... -c -tags testrunmain -o ./bin/local/ecs-cli.test ./ecs-cli
```

Comment:
would it be better to build the binary in the `build` stage as before, or is there some context here re: env vars and dirs that makes it better to execute here? (not sure if shell context changes between build phases)

Reply:
exported variables get cleared between phases but files persist. Because this generates a file it doesn't particularly matter where it happens. Happy to move it for consistency.